feat: add `math/base/special/sindf`

anandkaranubc opened the pull request:
Progresses https://github.com/stdlib-js/stdlib/issues/649

## Description

> What is the purpose of this pull request?

This pull request:

-  Adds the complete implementation for `math/base/special/sindf`

## Related Issues

> Does this pull request have any related issues?

This pull request:

-   Progresses https://github.com/stdlib-js/stdlib/issues/649

## Questions

> Any questions for reviewers of this pull request?

No.

## Other

> Any other information relevant to this pull request? This may include screenshots, references, and/or implementation notes.

No.

## Checklist

> Please ensure the following tasks are completed before submitting this pull request.

-   [x] Read, understood, and followed the [contributing guidelines][contributing].

* * *

@stdlib-js/reviewers

[contributing]: https://github.com/stdlib-js/stdlib/blob/develop/CONTRIBUTING.md

anandkaranubc committed:
feat: add `math/base/special/sindf`

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: passed
  - task: lint_package_json
    status: passed
  - task: lint_repl_help
    status: passed
  - task: lint_javascript_src
    status: passed
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: passed
  - task: lint_javascript_tests
    status: passed
  - task: lint_javascript_benchmarks
    status: passed
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: passed
  - task: lint_c_examples
    status: passed
  - task: lint_c_benchmarks
    status: passed
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: passed
  - task: lint_typescript_tests
    status: passed
  - task: lint_license_headers
    status: passed
---

anandkaranubc committed:
Update example.c

Signed-off-by: Karan Anand <119553199+anandkaranubc@users.noreply.github.com>

anandkaranubc committed:
Update README.md

Signed-off-by: Karan Anand <119553199+anandkaranubc@users.noreply.github.com>

Planeshifter commented on lib/node_modules/@stdlib/math/base/special/sindf/docs/types/index.d.ts:
```suggestion
* var v = sindf( 90.0 );
```

Planeshifter commented on lib/node_modules/@stdlib/math/base/special/sindf/package.json:
I realize these four keywords are also present in `sind`, but are they appropriate here? We do not have them in `sin` and I would expect them only for tangent/cotangent functions.

anandkaranubc committed:
docs: add missing space

Co-authored-by: Philipp Burckhardt <pburckhardt@outlook.com>
Signed-off-by: Karan Anand <119553199+anandkaranubc@users.noreply.github.com>

anandkaranubc committed:
Merge branch 'develop' into feat/sindf

anandkaranubc commented on lib/node_modules/@stdlib/math/base/special/sindf/package.json:
Thanks for the review, @Planeshifter! Making the changes here and also in `sind`.

anandkaranubc committed:
chore: remove irrelevant keywords

---
type: pre_commit_static_analysis_report
description: Results of running static analysis checks when committing changes.
report:
  - task: lint_filenames
    status: passed
  - task: lint_editorconfig
    status: passed
  - task: lint_markdown
    status: na
  - task: lint_package_json
    status: passed
  - task: lint_repl_help
    status: na
  - task: lint_javascript_src
    status: na
  - task: lint_javascript_cli
    status: na
  - task: lint_javascript_examples
    status: na
  - task: lint_javascript_tests
    status: na
  - task: lint_javascript_benchmarks
    status: na
  - task: lint_python
    status: na
  - task: lint_r
    status: na
  - task: lint_c_src
    status: na
  - task: lint_c_examples
    status: na
  - task: lint_c_benchmarks
    status: na
  - task: lint_c_tests_fixtures
    status: na
  - task: lint_shell
    status: na
  - task: lint_typescript_declarations
    status: na
  - task: lint_typescript_tests
    status: na
  - task: lint_license_headers
    status: passed
---

Planeshifter reviewed (approved):
Thanks, @anandkaranubc!
